storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst

```diff
@@ -414,107 +288,9 @@ database you can specify only the tables you want to restore:
 This command will download just ``ibdata1`` and ``sakila/payment.ibd`` table
 from the full backup.
```

Comment:
Where is the command that download this? Is it the one at line 100?

Reply:
Yes. Removed line on 415.